Resync css/css-typed-om WPT tests from upstream

cdumez opened the pull request:
#### ca1c4d4e3cc1e8d914ce56362a497f267c315785
<pre>
Resync css/css-typed-om WPT tests from upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=246655">https://bugs.webkit.org/show_bug.cgi?id=246655</a>

Reviewed by Tim Nguyen.

Resync css/css-typed-om WPT tests from upstream  5ae18d5d2a019ea8421b.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-normalization/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-objects/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/crashtests/cssTransform-Internal-value-expected.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/crashtests/cssTransform-Internal-value-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-subclasses/numeric-objects/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-subclasses/numeric-objects/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-subclasses/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/computed/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/declared/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/inline/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/text-transform.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em-expected.txt: Removed.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/255676@main">https://commits.webkit.org/255676@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ee4fe0dcb64b758542c2c51845fe798326926424

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/93204 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/2403 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/23854 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/102907 "Built successfully") | [❌ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/163195 "Found 2 new test failures: fast/css/aspect-ratio-min-height-replaced.html, fast/repaint/inline-outline-repaint.html (failure)") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/2409 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/30723 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/85593 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/99020 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/98875 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/1675 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/79672 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/28586 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/83588 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/83353 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/71696 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/37114 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/17232 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/34932 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/18474 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/38803 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/41005 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1823 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/40733 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/37684 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
r=me pending ews results

cdumez committed:
Resync css/css-typed-om WPT tests from upstream
https://bugs.webkit.org/show_bug.cgi?id=246655

Reviewed by Tim Nguyen.

Resync css/css-typed-om WPT tests from upstream  5ae18d5d2a019ea8421b.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-normalization/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-objects/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/crashtests/cssTransform-Internal-value-expected.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/crashtests/cssTransform-Internal-value-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-serialization/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-subclasses/numeric-objects/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-subclasses/numeric-objects/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-subclasses/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/computed/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/declared/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/inline/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/text-transform.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em-expected.txt: Removed.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/255676@main
